config/application.yml.default

```diff
@@ -327,49 +327,49 @@ production:
   database_worker_jobs_username: ''
   database_worker_jobs_host: ''
   database_worker_jobs_password: ''
-  disable_email_sending: 'false'
-  disallow_all_web_crawlers: 'false'
+  disable_email_sending: false
+  disallow_all_web_crawlers: false
   doc_auth_vendor: 'acuant'
-  doc_auth_vendor_randomize: 'false'
+  doc_auth_vendor_randomize: false
   doc_auth_vendor_randomize_percent: 0
   doc_auth_vendor_randomize_alternate_vendor: ''
   domain_name: login.gov
-  enable_test_routes: 'false'
-  enable_usps_verification: 'false'
+  enable_test_routes: false
+  enable_usps_verification: false
   hmac_fingerprinter_key:
   hmac_fingerprinter_key_queue: '[]'
-  logins_per_ip_limit: '20'
-  logins_per_ip_period: '20'
-  logins_per_ip_track_only_mode: 'true'
+  logins_per_ip_limit: 20
+  logins_per_ip_period: 20
+  logins_per_ip_track_only_mode: false
   newrelic_browser_app_id: ''
   newrelic_browser_key: ''
   newrelic_license_key: ''
   nonessential_email_banlist: '[]'
-  otp_delivery_blocklist_findtime: '5'
-  participate_in_dap: 'true'
+  otp_delivery_blocklist_findtime: 5
+  participate_in_dap: false
   password_pepper:
   piv_cac_verify_token_secret:
-  platform_authentication_enabled: 'false'
+  platform_authentication_enabled: false
   recurring_jobs_disabled_names: "[]"
   redis_throttle_url: redis://redis.login.gov.internal:6379/1
   redis_url: redis://redis.login.gov.internal:6379
-  report_timeout: 1_000_000
+  report_timeout: 1000000
```

Comment:
the integers with underscores are valid ruby and valid YAML, WDYT about keeping those?

Reply:
In identity_config, it is defined as integer, which is probably why it converts it. I think that we should probably be consistent with the predefined types? Any advantage/disadvantage to keeping it as is?

Reply:
Kk that's reasonable.